docs/concepts/accounts.md

```diff
@@ -0,0 +1,159 @@
+
+A simple representation of an account:
+
+![Screenshot 2023-08-16 at 16.43.07.png](_accounts_images/Screenshot_2023-08-16_at_16.43.07.png)
+
+**************Address**************
+
+The flow address is represented as 16 hex-encoded characters with a `0x` prefix. Unlike Bitcoin and Ethereum, Flow addresses are not derived from cryptographic public keys. Instead, each Flow address is assigned by the on-chain function that follows a deterministic addressing sequence.
+
+This decoupling allows for multiple public keys to be associated with one account, or for a single public key to be used across several accounts, and is a great strength of Flow.
+
+**************Balance**************
+
+Each account contains a Flow balance. This balance is used to pay for transaction fees and storage fees. More on that in the fees document.
+
+<aside>
+⚠️ The minimum amount of FLOW an account can have is **0.001**.
+
+</aside>
+
+This minimum is provided by the account creator at account creation. The minimum account reservation ensures that most accounts won't run out of storage capacity if anyone deposits anything (like an NFT) to the account. More on storage fees in the fees document.
+
+**********Contracts**********
+
+An account can optionally store multiple Cadence contracts. The code is stored as a human-readable UTF-8 encoded string which makes it easy for anyone to inspect the contents.
+
+**************Storage**************
+
+Contains all the stored resources in the account. These resources can be accessed by others in a secure and controlled manner by leveraging Cadence capabilities.
+
+## Account **********************Keys**********************
+
+Flow accounts can be configured with multiple public keys that are used to control access. Owners of the associated private keys can sign transactions to mutate the account's state.
+
+During account creation, we can provide public keys which will be used when interacting with the account. We can change the list of account keys by sending a transaction and signing it with current account keys, or we can even completely remove all the public keys from an account. This is radically different from blockchains like Ethereum.
+
+Each account key has a weight that determines the signing power it holds.
+
+<aside>
+⚠️ A transaction is not authorized to access an account unless it has a total signature weight greater than or equal to **1000**, the weight threshold.
+
+</aside>
+
+For example, an account might contain 3 keys, each with 500 weight:
+
+![Screenshot 2023-08-16 at 16.28.58.png](_accounts_images/Screenshot_2023-08-16_at_16.28.58.png)
```

Comment:
what do you mean? aren't the files in the folder with the docs?

Reply:
right, but in case we need to change the visuals, we'll need to create new ones if we don't have source code for them

Reply:
There is no "source code" for these images. We could buy a license for the program used to make them, but I don't think it's even worth it for this, because they won't change much or at all. And even if they change it's not hard to create a new image. It literally took me 2 min to create each image found here. It would be wise to use the same program tho https://excalidraw.com/ 
So maybe a good thing to add the link to this program in docs readme so if anyone wants to create images we all use it so it's consistent.

Reply:
you can save excalidraw diagram into a file

Reply:
nice, didn't know